src/spherexsphinx/conf/_utils.py

```diff
@@ -221,9 +221,9 @@ def set_edit_on_github(
         html_theme_options["use_edit_page_button"] = True
         html_context["github_user"] = github_owner
         html_context["github_repo"] = github_repo
-        html_context[
-            "github_version"
-        ] = self.config.project.github_default_branch
+        html_context["github_version"] = (
```

Comment:
What's the purpose of the parentheses?

Reply:
I think a change in Black caused the formatting to change.

Reply:
But it's not trying to create a tuple, right?  (In which case an extra comma would have been needed, I believe.)  It's just parenthesis-for-grouping?

Reply:
Correct; just parentheses for multi-line statement formatting, not for creating a tuple.